classes/controllers/grid/users/reviewer/PKPReviewerGridHandler.inc.php

```diff
@@ -467,6 +467,20 @@ function reviewRead($args, $request) {
 			$reviewAssignmentDao->updateObject($reviewAssignment);
 		}
 
+		if (!$reviewAssignment->getDateCompleted()) {
```

Comment:
Should this be here? This is triggered when the editor reads the review, not when they respond to it on the reviewer's behalf.